addon/components/frost-table-body.js

```diff
@@ -41,6 +42,15 @@ export default Component.extend({
 
   // == Computed Properties ===================================================
 
+  @readOnly
+  @computed('itemKey')
+  _eachItemKey (itemKey) {
+    if (itemKey) {
+      return itemKey
+    }
+    return '@index'
+  },
+
```

Comment:
Hmm actually, I'll have to make sure I can safely use itemKey instead of the targeted eachItemKey, it may bleed across purposes